flexget/plugins/list/sonarr_list.py

```diff
@@ -29,7 +29,12 @@ class SonarrSet(MutableSet):
             'include_data': {'type': 'boolean', 'default': False},
             'search_missing_episodes': {'type': 'boolean', 'default': True},
             'ignore_episodes_without_files': {'type': 'boolean', 'default': False},
-            'ignore_episodes_with_files': {'type': 'boolean', 'default': False}
+            'ignore_episodes_with_files': {'type': 'boolean', 'default': False},
+            'profile_id': {'type': 'number', 'default': 1},
+            'season_folder': {'type': 'boolean', 'default': False},
+            'monitored': {'type': 'boolean', 'default': True},
+            'root_folder_path': {'type': 'number', 'default': 1},
+            'series_type': {'type': 'string', 'default': 'standard'}
```

Comment:
This is a closed list right? Please limit it with an `enum`

Reply:
yes and done.